src/vega.schema.ts

```diff
@@ -292,6 +292,8 @@ export interface VgAxisBase {
    *
    * __Default value:__  derived from [axis config](config.html#axis-config)'s `tickSize` (`6` by default).
    *
+   * __Default value:__ `2/3*rangeStep` (This will provide offset between band equals to the width of the tick.)
```

Comment:
Still, you might wanna make sure that the default of all axis configs are not "derived from config.".  

For these properties that are both config and property, just state the default axis config.